src/routes/index.js

```diff
@@ -3,13 +3,13 @@ const request = require('request');
 const router = express.Router();
 const MarkdownIt = require('markdown-it');
 const md = new MarkdownIt();
+const FetchRelease = require('../../node_modules/fetchrelease/src/fetchrelease.js');
```

Comment:
Idk why, but it threw errors when i tried just `require('fetchrelease');`. Any idea why?

Reply:
ugh ugh nuuuuu, it hurts it hurts! You define main.js here: https://github.com/ubports/fetchrelease/blob/master/package.json#L5 but it does not exist. you need to use src/module.js

Reply:
Example: https://github.com/ubports/system-image-node-module/blob/master/package.json#L5